backend/src/main/java/com/github/gadenko/gamereviewsite/backend/security/repository/AppUserRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.github.gadenko.gamereviewsite.backend.security.repository;
+
+
+import com.github.gadenko.gamereviewsite.backend.security.model.AppUser;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+import java.util.Optional;
+
+@Repository
+public interface AppUserRepository extends MongoRepository<AppUser, String> {
+    Optional<AppUser> findByUsername(String username);
+}
```

Comment:
Wir hatten hier in der Shopping App noch die Methode fürs Dto drinnen gehabt:
Optional<AppUserItemDto> findAppUserByUsername(String username);

aber ich weiß nicht mehr genau, an welcher Stelle wir die gebraucht haben.